code/modules/power/lighting/lighting.dm

```diff
@@ -719,10 +733,10 @@ var/global/list/obj/machinery/light/alllights = list()
 /obj/item/weapon/light/tube/large
 	w_class = W_CLASS_SMALL
 	name = "large light tube"
-	brightness_range = 15
+	brightness_range = 8
 	brightness_power = 4
 	starting_materials = list(MAT_GLASS = 200, MAT_IRON = 100)
-	cost = 15
+	cost = 8
```

Comment:
did you map in tons of new light tubes and this is to offset that?

Reply:
I did before, but now maps are roughly identical now, and I forgot to it back. Good catch.